app/models/concerns/avatarable.rb

```diff
@@ -3,8 +3,10 @@ module Avatarable
 
   included do
     has_one_attached :avatar do |attachable|
-      attachable.variant :thumb, resize_to_limit: [64, 64]
-      attachable.variant :medium, resize_to_limit: [256, 256]
+      attachable.variant :thumb, resize_to_limit: [32, 32]
```

Comment:
Is there any risk in the variant being named the same thing but the size changed... so we would never see the correct variant size?

Reply:
from my test using a seeded data from #530 and then adding those variant it is working as expected but a double check could be useful indeed

Reply:
Nice. I'm going to run the backfill again on deploy, just in case.